src/sentry/integrations/vsts/integration.py

```diff
@@ -87,6 +87,16 @@ def get_unmigratable_repositories(self):
             external_id__in=[r['identifier'] for r in self.get_repositories()],
         )
 
+    def has_repo_access(self, repo):
+        client = self.get_client()
+        try:
+            # since we don't actually use webhooks for vsts,
+            # just verify repo access
```

Comment:
We do – they're just called Service Hooks/Subscriptions/it's confusing. https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/vsts/client.py#L247

Reply:
yeah i think we don't for commits, so assuming this will be ok 🤷‍♀️